ext/opcache/zend_accelerator_module.c

```diff
@@ -1010,3 +1017,24 @@ ZEND_FUNCTION(opcache_is_script_cached)
 
 	RETURN_BOOL(filename_is_in_cache(script_name));
 }
+
+ZEND_FUNCTION(require_modules)
```

Comment:
Does this mean that modules only work with opcache installed?

Reply:
That's right. As mentioned in the PR description, this is currently implemented in opcache for simplicity, but a proper implementation would move that somewhere else.

However, modules without a cache a slow to load (as seen in the "Wall time (cold)" benchmark results), and I would not recommend to use them without opcache.